src/Service.Tests/SqlTests/GraphQLQueryTests/GraphQLQueryTestBase.cs

```diff
@@ -1568,12 +1568,16 @@ public async Task GraphQLQueryWithMultipleOfTheSameFieldReturnsFieldOnce()
 
         #region Negative Tests
 
+        /// <summary>
+        /// This test checks the failure on providing invalid first parameter in graphQL Query.
+        /// We only allow -1 or positive integers for first parameter.-1 means max page size.
```

Comment:
is there a test that validates behavior when user provides `first:0`? 

I added some other comments calling out first:0 translates to default page size prior to this change. If we now error out, that 's a breaking change. would be worth having a test to either assert failure or ensure success (to be "regression detection").

Reply:
Yes see this from main: https://github.com/Azure/data-api-builder/blob/a23c590ceb45dbac416f9d038aa9d8a6e7d85317/src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs#L1548